src/meshapi/tests/test_nn.py

```diff
@@ -37,6 +37,7 @@ def setUp(self):
         self.building.save()
 
         inst = sample_install.copy()
+        inst["status"] = Install.InstallStatus.REQUEST_RECEIVED
```

Comment:
Add test that install numbers that’s used for their own node are set to pending, not NN assigned